db/migrations/sdp-migrations/2024-09-16.0-receiver-wallet-add-otp-confirmed-by-column.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+ALTER TABLE receiver_wallets
+    ADD COLUMN otp_confirmed_by VARCHAR(256) NULL;
```

Comment:
WDYT about `otp_confirmed_with`? The `by` suffix seems to refer to a person, while `with` would be a means.

I'm not so strongly on that mindset though so feel free to leave the naming as is.

Reply:
addressed in 212dd2c95c195c412e32f16ad31db1982a8701ed